dataland-frontend/tests/component/testUtils/SortTestUtils.ts

```diff
@@ -0,0 +1,60 @@
+import { ReportingPeriodOfDataSetWithId, sortReportingPeriodsToDisplayAsColumns } from "@/utils/DataTableDisplay";
+
+/**
+ * Calls the testfunction for sorting and swaps the list entries if necessary.
+ * @param  listOfDataDateToDisplayAsColumns list of objects to sort
+ * @param shouldSwap toogles the swap of both list elements in listOfDataDateToDisplayAsColumns (in case there are two.
+ * Shortens the test-function and avoids code duplications.
+ * @returns sorted list
+ */
+export function swapAndSortReportingPeriodsToDisplayAsColumns(
+  listOfDataDateToDisplayAsColumns: ReportingPeriodOfDataSetWithId[],
+  shouldSwap = false
+): ReportingPeriodOfDataSetWithId[] {
+  let swappedList: ReportingPeriodOfDataSetWithId[];
+  if (shouldSwap && listOfDataDateToDisplayAsColumns.length == 2) {
+    swappedList = listOfDataDateToDisplayAsColumns.slice();
+    swappedList[0] = listOfDataDateToDisplayAsColumns[1];
+    swappedList[1] = listOfDataDateToDisplayAsColumns[0];
+    listOfDataDateToDisplayAsColumns = swappedList.slice();
+  }
+  return sortReportingPeriodsToDisplayAsColumns(listOfDataDateToDisplayAsColumns);
+}
+
+/**
+ * @param firstYearObject first object used in sorting test
```

Comment:
No description and what is this parameter list? However ignore this comment until the other related comment is done. Maybe working on this one is not necessary anymore after that.

Reply:
removed p2p sorting test and therefore this file.
Lskg Test is complete now.